README.md

```diff
@@ -72,6 +73,8 @@ To set up LDR in this mode, provide a redis host and port, and supply a Redis ke
         prefix = "ld:spree:test"
         apiKey = "SPREE_TEST_API_KEY"
 
+You can also configure an in-memory cache for the relay to use so that connections do not always hit redis. To do this, set the `ttl` parameter in your `redis` configuration section to a number (in milliseconds). 
```

Comment:
I think we should have the in-memory cache's default TTL be > 0. Anyone running LD-Relay for scale reasons will likely want it enabled.